uwp/RichTextBox/Comment.md

```diff
@@ -87,3 +87,20 @@ private void RichTextBoxAdv_CommentAdding(object obj, CommentAddingEventArgs arg
 {% endtabs %}
 ![Comment_img2](Comment_images/Comment_img2.jpeg)
 
+The SfRichTextBoxAdv control allows you to determine the visibility of the comment pane using the IsCommentPaneVisible property of the EditorSettings class.
```

Comment:
Add the topic

Visibility of comment pane

Reply:
Added topic